src/features/game/expansion/LandExpansion.tsx

```diff
@@ -50,6 +51,10 @@ export const LandExpansion: React.FC = () => {
             // TODO dynamic game board size based on tile dimensions
           >
             <img src={background} className="absolute inset-0 w-full h-full" />
+            <div
+              className="absolute inset-0 bg-repeat w-full h-full"
+              style={{ backgroundImage: `url(${waterMovement})` }}
```

Comment:
If you look closely, the water appears quite blurry. Consider adding the pixelated image rendering to the background image.

It also appears that the water pixels are smaller than the land pixels. We need to get the water pixel sizes perfect from the beginning as we will use this to size a range of objects on the water.

Try to scale the water so it correctly matches the pixel size of the land.

<img width="410" alt="Screen Shot 2022-07-08 at 9 04 54 am" src="https://user-images.githubusercontent.com/11745561/177885802-021c5ff9-29d2-47c3-a47b-2f8e94071ab0.png">